src/env/node.ts

```diff
@@ -20,31 +20,37 @@ const { window: virtualWindow } = new JSDOM(
       FetchExternalResources: ['img'],
```

Comment:
I don't think this is needed, it seems to be a relic of the past

Reply:
it could still be needed to actually load base64 images.
Since canva is not mandatory for jsdom, loading images is completely optional.
That is my understanding, if you want them to actually be loaded, you have to ask for it

Reply:
I think it is dead.
Not typed and I searched the repo and couldn't find it.

Reply:
yes is not more there indeed.
It is replaced by resources 'usable' we have below.